integration/deploy/deploy_test.go

```diff
@@ -0,0 +1,114 @@
+//go:build integration
```

Comment:
Would it make sense to call this file `k8s_manifest_test.go`? the rest of files within this package makes reference to the kind/source of the app to be deployed